Gemfile

```diff
@@ -106,7 +106,7 @@ group :development, :test do
   gem 'rails-controller-testing'
   gem 'pry-byebug'
   gem 'guard-rspec', require: false
-  gem 'rubocop', '~> 1.47.0', require: false
+  gem 'rubocop', '~> 1.53.0', require: false
```

Comment:
i'm not sure what the implications of this are, if any! since we don't enforce use of rubocop i assume it's okay though

Reply:
Just fixing some local rubocop struggles for meeeee